plugins/plugin-bitbucket/codenvy-plugin-bitbucket-ext-bitbucket-server/src/main/java/org/eclipse/che/ide/ext/bitbucket/server/BitbucketURLTemplates.java

```diff
@@ -41,6 +41,11 @@ public String pullrequestUrl(String owner, String repositorySlug) {
     }
 
     @Override
+    public String updatePullrequestUrl(String owner, String repositorySlug, int pullrequestId) {
```

Comment:
Why we can't write `updatePullrequestUrl` in camel case here? I mean `updatePullRequestUrl`